src/controllers/requestController.js

```diff
@@ -58,13 +59,75 @@ class Requests {
   async getPendingApprovals(req, res) {
     try {
       const field = { status: 'Pending' };
-      const data = await RequestService.findByField(field);
+      const data = await requestService.findByField(field);
       const message = data.length > 0 ? 'Requests retrieved ' : 'No request pending approval';
 
       return Response.customResponse(res, 200, message, data);
     } catch (error) {
       return Response.errorResponse(res, 500, 'Something went wrong', error);
     }
   }
+
+  /**
+   * @param {object} req request
+   * @param {object} res response
+   * @return {function} requests
+   */
+  async rejectRequest(req, res) {
+    try {
+      const { reason } = req.body;
+      const manager = req.user;
+      const { requestId } = req.params;
+      const request = await requestService.getRequest({ id: requestId });
+      if (!request) {
+        return Response.errorResponse(res, 404, 'request not found', 'Not found');
+      }
+      const requesterId = request.user;
```

Comment:
check line 79 and line 85. I looks like a repetition assigning variables requst.user